tests/pages/pmmInventoryPage.js

```diff
@@ -81,6 +82,14 @@ module.exports = {
     return serviceIds;
   },
 
+  async getOtherDetails(serviceID, detailsSection, expectedResult) {
```

Comment:
what is the reasoning behind such a name of the function? 

Could you improve the function name to clearly represent what it is meant to do, also improve the assert text message, which details section didn't match

Reply:
Adding Service name to this failure assert makes more sense too

Reply:
![image](https://user-images.githubusercontent.com/63704168/116054938-82782700-a67c-11eb-9c29-17a8ec8e7a63.png)
The column is named 'Other Details' :) I am going to specify it better :)